### ISSUE TITLE
Spaces between nodes are not preserved

### ISSUE THREAD

**Issue by mcliment** [SebastianStehle/mjml-net#99]
First of all, thanks for this library as it makes our lives easier!

I've found an issue in the way whitespace is treated.

Given the text:

```html
<mj-text>Our <strong>MJML</strong> and <strong>friends</strong> are happy to announce: <strong>an awesome new library!</strong></mj-text>
```

The resulting text should not alter the HTML inside the `mj-text` element, but it does remove the space after the `<strong>` tags:

```html
<div style="...">Our <strong>MJML</strong>and <strong>friends</strong>are happy to announce: <strong>an awesome new library!</strong></div>
```

I've been doing some investigation an the problem is in the `RenderRaw` method as it does a `TrimXmlStart` for all the nodes (btw, `i` is not used in the loop) so it removes significant whitespace.

**Comment by SebastianStehle** [comment]
I see. The problem is that i is not incremented

**Comment by LiamRiddell** [comment]
@mcliment, thank you for the bug report. @SebastianStehle, I've created PR with a fix.

**Comment by SebastianStehle** [comment]
If you are happy with the library it would be great if would place your logo or so in the Readme.

**Comment by LiamRiddell** [comment]
Fixed by #100. Closing.

**Comment by mcliment** [comment]
@SebastianStehle Not sure about the logo... we're an energy seller that just operates in Spain, but if you need some logos I can ask for permission.
